chainio/clients/avsregistry/writer.go

```diff
@@ -843,6 +843,32 @@ func (w *ChainWriter) UpdateAVSMetadataURI(
 	return receipt, nil
 }
 
+// Removes strategies and their associated weights from the specified quorum.
+func (w *ChainWriter) RemoveStrategies(
+	ctx context.Context,
+	quorumNumber types.QuorumNum,
+	indicesToRemove []*big.Int,
+	waitForReceipt bool,
+) (*gethtypes.Receipt, error) {
+	w.logger.Info("removing strategies from quorum ", quorumNumber)
+
+	noSendTxOpts, err := w.txMgr.GetNoSendTxOpts()
+	if err != nil {
+		return nil, err
+	}
+
+	tx, err := w.stakeRegistry.RemoveStrategies(noSendTxOpts, quorumNumber.UnderlyingType(), indicesToRemove)
+	if err != nil {
+		return nil, err
+	}
+
+	receipt, err := w.txMgr.Send(ctx, tx, waitForReceipt)
+	if err != nil {
+		return nil, utils.WrapError("failed to remove strategies form quorum", err.Error())
```

Comment:
typo. Also, let's not add the `Error()` call. It just serializes the error.
```suggestion
		return nil, utils.WrapError("failed to remove strategies from quorum", err)
```